### DISCUSSION TITLE
Deploying Clickhouse Kafka Connect on Non-Confluent Connect Cluster

### DISCUSSION THREAD

**Discussion by erezblm** [ClickHouse/clickhouse-kafka-connect#156] [Q&A] [answered]
Hi,

I have a deployment of Kafka within Kubernetes using the Strimzi operator, and I wondered if I can deploy the Clickhouse Kafka Connector into the strimzi deployed Kafka Connect cluster.

I'm currently using standard open source connectors, and assumed that Clickhouse Connector would function in the same manner.

Regards,
Erez.

**Comment by Paultagoras** [comment, marked as answer]
The connector is meant to be a standard sink connector following the Kafka Connect standards, rather than any particular environment (like Confluent or MSK) - assuming that instance supports standard connectors (which it sounds like, if there are other connectors being used) it should work fine.

If you DO run into any issues though, please let us know!

**Reply by erezblm** [reply]
that's great news! thanks, let you know if I have any issues

Thanks.

**Reply by arkapravasinha** [reply]
I am running it issues, it is not able to run the connector

**Reply by arkapravasinha** [reply]
Can someone please help?

**Reply by Paultagoras** [reply]
> Can someone please help?

What issues are you running into?
